app/scripts/modules/core/src/overrideRegistry/Overridable.tsx

```diff
@@ -55,22 +55,28 @@ export function overridableComponent <P extends IOverridableProps, T extends Rea
       super(props);
 
       const { accountService } = ReactInjector;
+      let constructing = true;
```

Comment:
makes me feel super dirty, but i can't think of a better way to do this.

Reply:
yeah, think of a better way plz

Reply:
yeah chris and i have been discussing how gross this is, haven't come up with a better way yet.

on that note — can you drop a comment somewhere in here covering:

1) this is Bad(TM), and we should Do Better Next Time(TM)
2) why we couldn't Do Better Next Time, This Time(TM)